orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/monitoreddeploy/BroadcastDeployCompletedStage.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2019 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.orca.clouddriver.pipeline.monitoreddeploy;
+
+import com.netflix.spinnaker.orca.clouddriver.tasks.monitoreddeploy.BroadcastDeployCompletedTask;
+import com.netflix.spinnaker.orca.pipeline.StageDefinitionBuilder;
+import com.netflix.spinnaker.orca.pipeline.TaskNode;
+import com.netflix.spinnaker.orca.pipeline.model.Stage;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.stereotype.Component;
+
+@Component
```

Comment:
For these stages it might be helpful to include a short javadoc comment so that it's obvious what the stage is for

Reply:
absolutely, thanks, adding now